include/circular_buffer.hpp

```diff
@@ -468,7 +468,21 @@ namespace jm {
         {
             return JM_CB_ADDRESSOF(_buffer[0]._value);
         }
+        
+        JM_CB_CXX14_CONSTEXPR reference operator[](size_type i) JM_CB_NOEXCEPT
+        {
+            return _buffer[(_head + i) % N]._value;
+        }
 
+        JM_CB_CXX14_CONSTEXPR reference at(size_type i)
+        {
+            if (i >= N)
+            {
+                throw std::out_of_range("Array index error");
```

Comment:
Hey, sorry for not replying to your issue and thanks for the additions 👍 

However, could you change the exception message to `circular_buffer::at index out of range` to reflect the source of exception better?